vignettes/Vaccines.Rmd

```diff
@@ -26,170 +26,122 @@ We are going to set the default parameters to run the simulation from an equilib
 ```{r}
 year <- 365
 sim_length <- 3 * year
-human_population <- 1000
-starting_EIR <- 50
+human_population <- 10000
+starting_EIR <- 20
 
 simparams <- get_parameters(list(
     human_population = human_population,
-    model_seasonality = TRUE, # Let's try a bi-modal model
-    g0 = 0.28605,
-    g = c(0.20636, -0.0740318, -0.0009293),
-    h = c(0.173743, -0.0730962, -0.116019),
-    prevalence_rendering_min_ages = 0,
-    prevalence_rendering_max_ages = 18 * year,
     incidence_rendering_min_ages = 0,
-    incidence_rendering_max_ages = 18 * year,
+    incidence_rendering_max_ages = 5 * year,
     individual_mosquitoes = FALSE
   )
 )
 
 simparams <- set_equilibrium(simparams, starting_EIR)
 
-# Plotting functions
-plot_prevalence <- function(output) {
-  ggplot(output) + geom_line(
-    aes(x = timestep, y = (n_inc_0_6570 / n_0_6570))) + 
-    labs(x = "timestep", y = "incidence 0-18")
-}
-
-add_intervention_lines <- function(plot, events) {
-  plot + geom_vline(
-    data = events,
-    mapping = aes(xintercept=timestep),
-    color="blue"
-  ) + geom_text(
-    data = events,
-    mapping = aes(x = timestep, y = 0, label = name),
-    size = 4,
-    angle = 90,
-    vjust = -0.4,
-    hjust = 0
-  )
-}
 ```
 
 Then we can run the simulation for a variety of Vaccination strategies:
 
 ## Mass RTS,S
 
-This is a round of RTS,S vaccine for individuals between 5 - 17 months and a booster after 18 months; coverage of 80%; for 10 years:
+This is a round of RTS,S vaccine for individuals between 5 - 17 months followed by
```

Comment:
Below you parameterize differently. Maybe:  

"This is a round of RTS,S vaccine for individuals between 5 months and 10 years followed by a booster after 18 months."

Could also change min_wait to 0 since it doesn't apply here - there is only one round of vaccination.